src/main/java/org/bookstore/controller/AuthController.java

```diff
@@ -0,0 +1,38 @@
+package org.bookstore.controller;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.bookstore.dto.user.UserLoginRequestDto;
+import org.bookstore.dto.user.UserLoginResponseDto;
+import org.bookstore.dto.user.UserRegistrationRequestDto;
+import org.bookstore.dto.user.UserResponseDto;
+import org.bookstore.exceptions.RegistrationException;
+import org.bookstore.security.AuthenticationService;
+import org.bookstore.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Authentication", description = "Login and authorize endpoints")
+@RequestMapping("/auth")
+@RestController
+@RequiredArgsConstructor
+public class AuthController {
+    private final UserService userService;
+    private final AuthenticationService authenticationService;
+
+    @Tag(name = "Register", description = "Register a new user")
```

Comment:
```suggestion
    @Operation(summary = "Register", description = "Register a new user")
```
Better provide more informative messages